build/build-utils.js

```diff
@@ -31,11 +31,8 @@ function minifyFileTransform(file) {
       code += chunk.toString();
       next();
     },
-    final(next) {
-      const result = terser.minify(code);
-      if (result.error) {
-        throw result.error;
-      }
+    async final(next) {
+      const result = await terser.minify(code, {ecma: 2019});
```

Comment:
not sure why updating to the latest terser isn't working.. https://github.com/terser/terser/pull/326

Reply:
oh, error must be in browserify, not terser.

Reply:
will node 12 finally force us to move to webpack/rollup/something other than browserify? :)